packages/botonic-core/src/core-bot.ts

```diff
@@ -37,20 +63,26 @@ export class CoreBot {
     this.router = isFunction(this.routes)
       ? null
       : new Router(
+          // @ts-ignore
           [...this.routes, ...this.defaultRoutes],
           this.inspector.routeInspector
         )
   }
 
-  getString(stringID, session) {
+  getString(stringID: string, session: Session): string {
```

Comment:
When it is typed, do we need to call it `stringId`? I'd say it can be just `id: string`